### PR TITLE
remove ...$$props

### PR DESCRIPTION
**Pull request by dmaevsky**

### CODE REVIEW

File: components/Input/Input.svelte [hunk 1]
```diff
@@ -8,6 +9,14 @@
   export let outlined = false;
 
   $: hidden = ((size * 12) - ((value || '').length + (label || '').length)) < -1;
+
+  const handleInput = e => {
+    // TODO: any other types here?
+    value = type.match(/^(number|range)$/)
```
Comment (matyunya) [inline components/Input/Input.svelte]: phone, email, password etc.
https://www.w3schools.com/html/html_form_input_types.asp

**Comment by dmaevsky** [comment]
But do we need any special treatment for those? They normally have a
built-in validation already

**Comment by matyunya** [comment]
Then why add special treatment for these two? There are dates, bools, files etc.

**Comment by dmaevsky** [comment]
That was a copy-paste from Rich's SO answer (
https://stackoverflow.com/questions/57392773/error-type-attribute-cannot-be-dynamic-if-input-uses-two-way-binding
).
Just to coerce the value to a number. Dates and files most probably remain
strings. Not sure about bools.
The question is: do we need anything else right now?

**Comment by matyunya** [comment]
I don't think we need these two neither, rather propagate on:input.

**Comment by dmaevsky** [comment]
If you're ok with 'numerical' input actually outputting a string

**Comment by dmaevsky** [comment]
But well, not that I really care :)

**Comment by matyunya** [comment]
I think they all output strings by default (same as dates, or checkboxes), so maybe better let user decide. That's why I stumbled with Smelte because there's no `on:*`

**Pull request merged**